HPXMLtoOpenStudio/resources/meta_measure.rb

```diff
@@ -4,21 +4,21 @@
 
 # TODO
 #
-# @param rundir [TODO] TODO
-# @param measures [TODO] TODO
-# @param measures_dir [TODO] TODO
-# @param debug [TODO] TODO
-# @param output_vars [TODO] TODO
-# @param output_meters [TODO] TODO
-# @param run_measures_only [TODO] TODO
-# @param print_prefix [TODO] TODO
-# @param ep_input_format [TODO] TODO
-# @param skip_simulation [TODO] TODO
-# @param suppress_print [TODO] TODO
-# @return [TODO] TODO
-def run_hpxml_workflow(rundir, measures, measures_dir, debug: false, output_vars: [],
-                       output_meters: [], run_measures_only: false, print_prefix: '',
-                       ep_input_format: 'idf', skip_simulation: false, suppress_print: false)
+# @param rundir [String] The run directory containing all simulation output files
+# @param measures [Hash] Map of OpenStudio-HPXML measure directory name => Array<Hash of measure arguments>
```

Comment:
Do you remember why it's `Array<Hash of measure arguments>` and not just `Hash of measure arguments`? I wonder if the use case for using an Array no longer applies and we could simplify this.

Reply:
I do not. Yeah that crossed my mind as well.

Reply:
I suggest we change it, I'm not immediately seeing why it's needed. You could then simplify run_simulation.rb. In fact, I think it would mean that you could get rid of the use of `update_args_hash` in run_simulation.rb, at which point I think the method could be removed from meta_measure.rb entirely.

Reply:
We've decided to remove the `update_args_hash` method, but keep the `measures = {"MeasureA": [{"arg1": "foo", "arg2": "bar", ...}, ...]}` structure. We need, for example, to be able to apply ReportSimulationOutput more than once in an `apply_measures` call.